mujina-idp/readme.md

```diff
@@ -0,0 +1,27 @@
+# Build the mock IdP docker image
+```bash
+docker build . -t laa-saml-mock-idp
+```
+
+# Run the mock IdP docker container
+```bash
+docker container run -p 8080:8080 laa-saml-mock-idp
```

Comment:
Maybe neater to omit container from the Docker run commands?
docker run -p 8080:8080 laa-saml-mock-idp

Reply:
The new standard is to use _docker container run_, __docker run__ will be deprecated in the future
- https://docs.docker.com/engine/reference/commandline/container_run/

Reply:
Great, thanks